src/kakarot/interfaces/interfaces.cairo

```diff
@@ -1,6 +1,7 @@
 %lang starknet
 
 from starkware.cairo.common.uint256 import Uint256
+from utils.utils import Option
```

Comment:
maybe put this in model as all the other type def?

Reply:
done